.github/CODEOWNERS

```diff
@@ -7,4 +7,4 @@
 # offer a reasonable automatic best-guess
 
 # catch-all rule (this only gets matched if no rules below match)
-*    @guolinke @jameslamb @shiyu1994 @jmoralez
+*    @guolinke @jameslamb @shiyu1994 @jmoralez @borchero
```

Comment:
@borchero this will automatically add you as a reviewer on every PR.

To be sure we set the right expectation...you do not need to review every PR! We require 1 approval from a maintainer to merge. In some situations, like more controversial changes or changes where particular maintainers have specialized knowledge, we sometimes ask for multiple approvals.

For example, for any non-trivial changes to distributed training, I'll probably ask for approvals from both @jmoralez (Dask knowledge) and one of @shiyu1994 and @guolinke (C++ and LightGBM internals knowledge).